SAMtags.tex

```diff
@@ -66,6 +66,7 @@ \section{Standard tags}
   {\tt CT} & Z & Complete read annotation tag, used for consensus annotation dummy features.\\
   {\tt E2} & Z & The 2nd most likely base calls \\
   {\tt FI} & i & The index of segment in the template \\
+  {\tt FT} & Z & Filter field, indicating the reason why this record has been modified\\ 
```

Comment:
I'm not sure what it means here `has been modified`.  Is that supposed to say `has been filtered`?